### ISSUE TITLE
Mask2Former pretraining script

### ISSUE THREAD

**Issue by yeshwanth95** [3dv-casia/PolyRoom#4]
Hi, thank you for releasing your work.
I'm presently trying to train your method on a custom dataset. Could you kindly share the Mask2Former training script you used to pretrain the room query initialization module on each dataset? It would save me a lot of time if you could share your training script.

**Comment by PaulLiuYZ** [comment]
Please refer to the https://mmdetection.readthedocs.io/en/latest/user_guides/train.html for user guides for training of  MMdetection, and I will later update the training script.

**Comment by PaulLiuYZ** [comment]
python tools/train.py configs/str3d/mask-rcnn_r50-caffe_fpn_ms-poly-1x_str3d.py
